runtime/command-start/src/main/java/io/aklivity/zilla/runtime/command/start/internal/airline/ZillaStartCommand.java

```diff
@@ -64,28 +65,26 @@ public final class ZillaStartCommand extends ZillaCommand
     public int workers = -1;
 
     @Option(name = {"-P", "--property"},
-        description = "Property name=value",
-        hidden = true)
+        description = "Property name=value")
     public List<String> properties;
 
     @Option(name = {"-p", "--properties"},
-        description = "Path to properties",
-        hidden = true)
+        description = "Path to properties")
     public String propertiesPath;
 
-    @Option(name = "-e",
-        description = "Show exception traces",
-        hidden = true)
+    @Option(name = {"-e", "--exception-traces"},
+        description = "Show exception traces")
     public boolean exceptions;
 
     @Override
     public void run()
     {
         Runtime runtime = getRuntime();
         Properties props = new Properties();
-        props.setProperty(ENGINE_DIRECTORY.name(), ".zilla/engine");
+        String directory = System.getProperty(ZILLA_DIRECTORY_PROPERTY, ".");
+        props.setProperty(ENGINE_DIRECTORY.name(), String.format(ZILLA_ENGINE_PATH_DEFAULT, directory));
 
-        Path path = Paths.get(propertiesPath != null ? propertiesPath : OPTION_PROPERTIES_PATH_DEFAULT);
+        Path path = Paths.get(propertiesPath != null ? propertiesPath : String.format(OPTION_PROPERTIES_PATH_DEFAULT, directory));
```

Comment:
`directory` can be moved to a constant `ZILLA_DIRECTORY`, and the other constants can be derived in place using String.format(...) in each constant definition, so no need to change the run method except to introduce `ZILLA_ENGINE_PATH_DEFAULT`.